api/v1/questions.json

```diff
@@ -0,0 +1,29 @@
+---
+layout: null
+---
+[
+  {%- for content in site.content %}
+  {%- assign category = site.categories | where: 'name', content.category | first %}
+  {
+    "question_url": "{{ site.url }}/{{ content.category | downcase }}/{{ content.slug | downcase }}/",
+    "title": "{{ content.title | replace: '"', '\"' }}",
+    "answer": "{{ content.content | strip_html | strip_newlines | remove_chars | escape }}",
+    "answer_html": "{{ content.content | strip_newlines | replace: '"', '\"' }}",
+    "updated_at": "{{ content.date | date: '%Y-%m-%d' }}",
+    "category": {
+      "title": "{{ category.title }}",
+      "category_url": "{{ site.url }}/{{ category.name | downcase }}/"
+    },
+    "sources": [
+      {
+        "name": "{{ content.source | replace: '"', '\"' }}",
+        "source_url": "{{ content.source_url }}"
+      }{% if content.source_2 %},
+      {
+        "name": "{{ content.source_2 | replace: '"', '\"' }}",
```

Comment:
FYI...the `source_2` key no longer exists. This was [converted to an array in this PR](https://github.com/18F/cv_faq/pull/523), which maps to agency markdown files in the `_agencies` directory.